File-Formats/PDF/Convert-HTML-To-PDF/azure.md

```diff
@@ -318,133 +301,109 @@ public static async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Func
+   {
+   FileAccessPermissions ExecutableFilePermissions = FileAccessPermissions.UserRead | FileAccessPermissions.UserWrite | FileAccessPermissions.UserExecute |
+   FileAccessPermissions.GroupRead | FileAccessPermissions.GroupExecute | FileAccessPermissions.OtherRead | FileAccessPermissions.OtherExecute;
+   string[] executableFiles = new string[] { "chrome", "chrome_sandbox" };
+   foreach (string executable in executableFiles)
+   {
+       var execPath = Path.Combine(tempBlinkDir, executable);
+       if (File.Exists(execPath))
+       {
+          var code = Function1.Chmod(execPath, ExecutableFilePermissions);
+           if (code != 0)
+           {
+               throw new Exception("Chmod operation failed");
+           }
+       }
     }
-}
 
-{% endhighlight %}
+       {% endhighlight %}
 
-Include the below enum in the Function1.cs file. 
+7. Include the below enum in the Function1.cs file. 
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-[Flags]
-internal enum FileAccessPermissions : uint
-{
-OtherExecute = 1,
-OtherWrite = 2,
-OtherRead = 4,
-
-GroupExecute = 8,
-GroupWrite = 16,
-GroupRead = 32,
-
-UserExecute = 64,
-UserWrite = 128,
-UserRead = 256
-}
+   [Flags]
+   internal enum FileAccessPermissions : uint
+   {
+   OtherExecute = 1,
+   OtherWrite = 2,
+   OtherRead = 4,
+   GroupExecute = 8,
+   GroupWrite = 16,
+   GroupRead = 32,
+   UserExecute = 64,
+   UserWrite = 128,
+   UserRead = 256
+   }
 
-{% endhighlight %}
+   {% endhighlight %}
 
-### Publish to Azure Functions Linux
+## Publish to Azure Functions Linux
 
-Right-click the project and select Publish. Then, create a new profile in the Publish Window. The Blink rendering engine will work in consumption plan. So, you can create the Azure Function App service with a consumption plan.
+1. Right-click the project and select Publish. Then, create a new profile in the Publish Window. The Blink rendering engine will work in consumption plan. So, you can create the Azure Function App service with a consumption plan.
 ![Convert HTMLToPDF Azure Functions Step4](htmlconversion_images/AzureFunctions4.png)
 
-After creating the profile, click the Publish button.
+2. After creating the profile, click the Publish button.
 ![Convert HTMLToPDF Azure Functions Step5](htmlconversion_images/AzureFunctions5.png)
 
-Now, go to the Azure portal and select the App Services. After running the service, click Get function URL > Copy. Include the URL as a query string in the URL. Then, paste it into the new browser tab. You will get the PDF document as follows.
+3. Now, go to the Azure portal and select the App Services. After running the service, click Get function URL > Copy. Include the URL as a query string in the URL. Then, paste it into the new browser tab. You will get the PDF document as follows.
 Example: https://htmltopdfazurefunctionblink.azurewebsites.net/api/Function1?code=aYCeymbJeHMGIwEEIuCIcNe….&url=https://www.syncfusion.com
```

Comment:
Please make it as "italic" style for example

Reply:
Example like is not working